### PR TITLE
#RI-2666 - update key header, add full screen mode

### PR DESCRIPTION
**Pull request by romansergeenkosofteq**

### CODE REVIEW

File: redisinsight/ui/src/pages/browser/components/key-details/KeyDetailsWrapper.tsx [hunk 1]
```diff
@@ -17,17 +17,26 @@ import { refreshListElementsAction } from 'uiSrc/slices/list'
...
+  onToggleFullScreen: () => void
+  onCloseKey: () => void
+  onEditKey: (key: string, newKey: string) => void
+  onDeleteKey: () => void
+  keyProp: string | null
 }
 
-const KeyDetailsWrapper = ({ onCloseKey, onEditKey, onDeleteKey, keyProp }: Props) => {
+const KeyDetailsWrapper = (
+  {
```
Comment (zalenskiSofteq) [inline redisinsight/ui/src/pages/browser/components/key-details/KeyDetailsWrapper.tsx:29]: Destructuring from *props* variable pls, not inside round brackets

File: redisinsight/ui/src/slices/app/info.ts [hunk 2]
```diff
@@ -7,7 +7,7 @@ import { GetServerInfoResponse } from 'apiSrc/dto/server.dto'
 import { AppDispatch, RootState } from '../store'
 import { StateAppInfo } from '../interfaces'
 
-const segmentWriteKey = process.env.SEGMENT_WRITE_KEY || 'SOURCE_WRITE_KEY'
+const segmentWriteKey = 'KSKD2KYXyEC2298VjgAKKDC0pwCgMc1q'
```
Comment (zalenskiSofteq) [inline redisinsight/ui/src/slices/app/info.ts:10]: remove pls

File: redisinsight/ui/src/pages/browser/components/key-details/KeyDetailsWrapper.tsx (hunk 1 above)
Comment (romansergeenkosofteq) [reply redisinsight/ui/src/pages/browser/components/key-details/KeyDetailsWrapper.tsx]: done

File: redisinsight/ui/src/slices/app/info.ts (hunk 2 above)
Comment (romansergeenkosofteq) [reply redisinsight/ui/src/slices/app/info.ts]: removed, thanks :D

**Review by zalenskiSofteq — APPROVED** [review]

**Pull request merged**